crates/nu-command/src/filesystem/cp.rs

```diff
@@ -158,7 +157,25 @@ impl Command for Cp {
 
                 for (src, dst) in sources {
                     if src.is_file() {
-                        let res = if interactive && dst.exists() {
+                        #[cfg(not(windows))]
+                        let dst = match std::fs::canonicalize(dst.clone()) {
```

Comment:
using `nu_path::canonicalize_with` instead of `std::fs::canonicalize` should make everything works fine (which because `nu_path::canonicalize_with` use `dunce::canonicalize` internal, and it works specially for windows path.

```rust
let dst = match canonicalize_with(dst.clone(), &current_dir_path) {
    Ok(path) => path,
    Err(_) => dst,
};
```

Reply:
Thanks guys, much appreciated! I will try this tomorrow :).

BTW, I took a quick look at crate dunce - [Filesystem paths in Windows are a total mess](https://docs.rs/dunce/1.0.1/dunce/)